wordpress-mobile/WordPress-iOS
# Modernize the push notification sound

Fixes #9594 

Updates the sound file for push notifications to something relatively similar in nature of a higher quality.

To test:

1. Install on a device.
2. Make sure a dev push notification server is up and running in someone's WordPress.com developer sandbox.
3. Send a push notification to your device.
